machine_test.go

```diff
@@ -49,9 +49,11 @@ const (
 var testDataPath = "./testdata"
 
 var skipTuntap bool
+var rootEnabled bool
 
 func init() {
 	flag.BoolVar(&skipTuntap, "test.skip-tuntap", false, "Disables tests that require a tuntap device")
+	flag.BoolVar(&skipTuntap, "test.root", false, "Enables tests that require root")
```

Comment:
Can you reverse this?  I'd much prefer to have root enabled by default and disabled explicitly so that anyone who *is* running without root knows that they're skipping tests.